### DISCUSSION TITLE
Is it possible to check for collisions with object sprites?

### DISCUSSION THREAD

**Discussion by 2Red** [Annoraaq/grid-engine#187] [Q&A]
I have a game idea where the character will get stuck in a rock wall or a tree. When the character uses the skill to destroy trees, he can move forward. Don't know if your plugin supports that or not?

Thanks.

**Comment by Annoraaq** [comment]
Yes. One way to achieve this would be to listen to the `positionChangeStarted()` observable and if the character steps on a tile that should trap her, you can simply not fire the `move()` method anymore until she destroys the obstacle.
